components/styles/cart.css

```diff
@@ -0,0 +1,143 @@
+.main-section-cart {
+  display: flex;
+  color: black;
+  position: relative;
+}
+
+#cart-details {
+  box-shadow: 0 1px 8px 0 rgb(0 0 0 / 6%);
+  color: black;
+  margin: 0.5rem;
+  display: block;
+  width: auto;
+  margin: 1rem;
+  padding: 1rem;
+  flex-shrink: 1;
+}
+
+#cart-details .product-cart {
+  display: flex;
+  width: 50rem;
+  border-bottom: 1px solid #e0e0e0;
+  margin: 1rem 0;
+  gap: 1rem;
+  padding-bottom: 1rem;
+}
+
+#cart-details .product-cart .desc {
+  flex-grow: 1;
+}
+
+#cart-details .product-cart .desc p:nth-child(2) {
+  color: grey;
+}
+
+#cart-details .product-cart img {
+  height: 10rem;
+  width: 10rem;
+}
+
+.product-cart .add-more {
+  display: flex;
+  justify-content: space-evenly;
+  margin-top: 1rem;
+}
+
+.product-cart .add-more .minus, .product-cart .add-more .plus {
+  height: 2rem;
+  width: 2rem;
+  border-radius: 50%;
+  text-align: center;
+  border: 1px solid grey;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+}
+
+.minus:hover, .plus:hover {
+  cursor: pointer;
+}
+
+.product-cart .add-more .number {
+  height: 2rem;
+  width: 3rem;
+  border: 1px solid grey;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.minus-disable {
+  color: #c2c2c2;
+  border-color: #e0e0e0;
```

Comment:
can use variables for colors

Reply:
I've used different colors in different branches, and thus will change them to variables in final merge.